src/Common/src/Microsoft/Win32/SafeHandles/SafeCloseIcmpHandle.cs

```diff
@@ -1,9 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.Win32.SafeHandles;
```

Comment:
It's returned by methods in the Common interop files.  Seemed strange to have the shareable things using something only available in a specific library, so I wanted to move it to either where we have general safe handle or to interop.  Reasonable?